raiden/log_config.py

```diff
@@ -252,4 +252,5 @@ def configure_logging(
 
     # set logging level of the root logger to DEBUG, to be able to intercept
     # all messages, which are then be filtered by the `RaidenFilter`
-    structlog.get_logger('').setLevel('DEBUG')
+    structlog.get_logger('').setLevel(logger_level_config.get('', DEFAULT_LOG_LEVEL))
+    structlog.get_logger('raiden').setLevel('DEBUG')
```

Comment:
default is ok for Raiden too - the debug can be found in the log file

Reply:
That's why this change: we need to set `DEBUG` at the logger level, to be able to catch everything on logger (and later **filter** separately for the debug file and standard **handlers**). So, before, everything was always `DEBUG`, so the handlers filters were the only responsible for doing this filtering. But, as pytest captures every logging call, it spammed travis logs with debug logs of all modules. This change still keeps the filters working on the handlers level, but also sets **some** level to the root logger (taking into account if user wants root to be debug too, with `--log-config`), to avoid travis getting overwhelmed. IOW: this sets the logger levels to `logger_level_config` _less-restrict_ levels, while still catching what needed for the main filter.